docs/developer-guide/lighting.md

```diff
@@ -0,0 +1,42 @@
+# Lighting
+
+Luma.gl supports different shading modules.
+To make 
+
+
+## Light Sources
```

Comment:
Give JS code examples of how to create a model with a material and set lighting uniforms... explain that we have two Material systems that both support the same lights...